MultiSource/Benchmarks/Makefile

```diff
@@ -43,4 +43,11 @@ PARALLEL_DIRS := $(filter-out PAQ8p, $(PARALLEL_DIRS))
 PARALLEL_DIRS := $(filter-out SciMark2-C,$(PARALLEL_DIRS))
 endif
 
+# The following tests fail after the upgrade to Clang 11.
```

Comment:
TODO: try to disable more specific directories, e.g. Olden/bh rather than Olden.